### DISCUSSION TITLE
Repository for heuristics for different websites frameworks

### DISCUSSION THREAD

**Discussion by gwpl** [Skyvern-AI/skyvern#1115] [Ideas]
There are some websites framework that are popular , so implementing heuristics or "tools" for those could be helpful.

Maybe if would be organized as separate repository community could contribute and Skyvern benefit?
 
(inspired by: https://github.com/Skyvern-AI/skyvern/issues/1114 )

**Comment by suchintan** [comment]
Great idea. We're gonna work on this soon!
